source/agora/consensus/validation/Block.d

```diff
@@ -615,10 +622,26 @@ unittest
     foreach (ref tx; GenesisBlock.txs)
         utxo_set.put(tx);
 
-    auto txs_1 = makeChainedTransactions(gen_key, null, 1,
-        400_000_000_000 * 8).sort.array;
```

Comment:
Does the `spend_amount` here matter for this unittest?
If not, we can use the default amount and the loop can be deleted.

Reply:
It doesn't matter - this test is specifically testing freeze transactions as far as I can tell.